index.html

```diff
@@ -216,8 +216,9 @@ <h1>Contact me</h1>
                         <input type="email" name="email" required id="email" placeholder="Your email...">
                         <label class="hidden-element" for="comment">Comment</label>
                         <textarea name="comment" id="comment" required cols="30" rows="10" placeholder="Your comments..."
-                            maxlength="500"></textarea>
-                         <input type="submit" value="Get in touch">   
+                        maxlength="500"></textarea>
+                        <input type="submit" class="btnSubmit" value="Get in touch">
```

Comment:
- [ ] [OPTIONAL] Good job so far 👏 I would suggest that you add the CSS property 
```css
.btnSubmit {
...
cursor: pointer;
}
```
so that user will know that he/she is about to click on clickable button 😉